src/Microsoft.Azure.WebJobs.Host/Queues/Listeners/HostMessageListenerFactory.cs

```diff
@@ -83,16 +80,9 @@ public Task<IListener> CreateAsync(CancellationToken cancellationToken)
         {
             ITriggerExecutor<IStorageQueueMessage> triggerExecutor = new HostMessageExecutor(_executor, _functionLookup, _functionInstanceLogger);
 
-            TimeSpan configuredMaximum = _queueConfiguration.MaxPollingInterval;
-            // Provide an upper bound on the maximum polling interval for run/abort from dashboard.
```

Comment:
So you're removing this upper bound here. Previously we were bounding this. Might not be an issue - what was your thought here?